hkmc2/shared/src/test/mlscript/backlog/UCS.mls

```diff
@@ -81,18 +81,10 @@ if
 let audits = new Set()
 //│ audits = Set{}
 
-:fixme
 if audits.has(1) === true do
```

Comment:
This fixed test case can now be moved out of the backlog into a normal test file (pick a meaningful one that maybe already tests `do`).